common/build.gradle

```diff
@@ -0,0 +1,18 @@
+architectury {
+	common(rootProject.enabled_platforms.split(","))
+}
+
+dependencies {
+	// We depend on fabric loader here to have `org.objectweb.asm` and `org.spongepowered.asm`
+	// Do NOT use other classes from fabric loader
```

Comment:
is it possible to only depend on `org.spongepowered.asm`

Reply:
the Architectury template I based this on also depends on fabric loader in the common module, but they do it for the `@Environment` annotation. I only edited the comment there because in this case we need those two packages which are also available in both fabric and forge environments, but we need to tell gradle that so we can use it in the common module. If you really want me to change that, I'll see what I can do